bin/env2options

```diff
@@ -78,17 +89,20 @@ def parse_args():
 def main():
     args = parse_args()
     if args.source == SOURCE_ALL:
-        values = get_merged_settings()
+        job_cmd = get_job_cmd()
+        merged_settings = get_merged_settings()
```

Comment:
@dangra I've just noticed that shub settings are merged [incorectly](https://github.com/scrapinghub/scrapinghub-image-casperjs/blob/master/bin/env2options#L25-L33) - it assumes there's a dict with string keys and dict values and merges it one by one in alphabetical order. 

While in real, 1) there should be strict order of merging, cause `job_settings` will be overwritten with `project settings`, etc 2) not all values are dicts - I guess we agreed to drop addons and `addons_settings` for non-Scrapy, but there're some others. 

I wonder if there was a discussion about it already, or there're some agreed assumptions I'm missing? We probably should hard-code settings section names somewhere and sort it properly.